script/testing/kill_server_on_port.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
```

Comment:
Can you rename this to `kill_server.py`?

Reply:
done